mlir/lib/Dialect/Rock/Tuning/GridwiseGemmParams.cpp

```diff
@@ -274,7 +204,8 @@ LogicalResult PopulateParams::obtainTuningParameters(
 }
 
 std::vector<InitParamsNonXDL>
-PopulateParams::getTuningParameters(ConvOpType dir, Type dataType) const {
+PopulateParams::getTuningParameters(Optional<ConvOpType> dir,
```

Comment:
Here also, we can drop the dir as it is not being used.

Reply:
Looks like we can indeed drop the direction. But I'm catching up here... we used to initialize that heuristic parameters based different direction as well. I seem to notice that `orderInitPrams()` does its job on deciding based on amount of padding. Do we do any special ordering based on conv direction as well?

Thinking out loud: It might be that we used a consistent set of heuristic init parameters against all 3 conv directions, which is how you are able to simplify the logic here?

An orthogonal topic, it looks like xdlops and nonxdlops has some unevenness on int8, and I suspect we'd need to bring over a different set of tuning parameters on int8 nonxdlops as well. It may be a rocm x.x ticket.